src/components/ColumnTwentyFourHourChange/index.js

```diff
@@ -0,0 +1,29 @@
+import { Arrow } from 'components/Arrow'
+import { Cell } from 'components/Cell'
+import { Column } from 'components/Column'
+import { getArrow } from 'utils/getArrow'
+import { keyGen } from 'utils/keyGen'

```

Comment:
same

Reply:
Refactored all folders / components / imports to follow this.